ground/gcs/src/plugins/coreplugin/mainwindow.cpp

```diff
@@ -181,14 +181,28 @@ MainWindow::MainWindow() :
     registerDefaultContainers();
```

Comment:
Yup, so everything in the stylesheet is subject to the QWidget layout stuff, e.g. size policies etc. 😁